tensorflow_io/json/kernels/json_input.cc

```diff
@@ -17,22 +17,32 @@ limitations under the License.
 #include <fstream>
 #include "kernels/dataset_ops.h"
 #include "tensorflow/core/lib/io/buffered_inputstream.h"
+#include "tensorflow/core/platform/env.h"
 #include "include/json/json.h"
 
 namespace tensorflow {
 namespace data {
 
 class JSONInputStream {
 public:
-  explicit JSONInputStream(io::InputStreamInterface* s, const string& filename)
-    : ifs_(filename){}
+  explicit JSONInputStream(io::InputStreamInterface* s, const string& filename) {
```

Comment:
Since `io::InputStreamInterface* s, ` is not going to be used any more, can you remove this parameter?

Reply:
Yes, PR updated